src/main/java/org/jabref/gui/fieldeditors/contextmenu/ClearField.java

```diff
@@ -0,0 +1,18 @@
+package org.jabref.gui.fieldeditors.contextmenu;
+
+import javafx.scene.control.MenuItem;
+import javafx.scene.control.TextArea;
+
+import org.jabref.logic.formatter.bibtexfields.NormalizeNamesFormatter;
+import org.jabref.logic.l10n.Localization;
+
+class ClearField extends MenuItem {
+
+    private final TextArea opener;
```

Comment:
I think there is no need for this private field since the variable is only used in the constructor anyway.